tests/test_parse.py

```diff
@@ -114,18 +113,22 @@ class MyModel(BaseModel):
     assert isinstance(m.__root__.__root__[0], str)
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_json():
-    assert Model.parse_raw('{"a": 12, "b": 8}') == Model(a=12, b=8)
+    # TODO: Are we planning to keep model_validate_json?
+    assert Model.model_validate_json('{"a": 12, "b": 8}') == Model(a=12, b=8)
```

Comment:
Question 6: Since we dropped all the other forms of parse_raw, just wanted to confirm that we want to keep model_validate_json around

Reply:
Isn't `model_validate_json` the "right" way to do this in V2?

Reply:
yes.